Pipfile

```diff
@@ -25,6 +25,8 @@ markupsafe = "==2.1.2"
 sqlalchemy = "==1.4.46"
 werkzeug = "==2.2.2"
 wtforms = "==3.0.1"
+importlib-metadata = "==6.9.0"
```

Comment:
Where are these 2 new packages being used? Were they installed by mistake?
(referencing importlib-metadata and zipp)

Reply:
Actually not sure how those are there.

Reply:
Both import were included in the starter files for the project